docs/development.rst

```diff
@@ -380,76 +380,38 @@ The following examples show to extend the images.
 
   The images ship with Azul Zulu OpenJDK.  Due to licensing restrictions, we cannot bundle Oracle JDK, but we are testing on Zulu OpenJDK and do suggest it as a viable alternative.  In the event that you really need to use Oracle's version, you can follow the steps below to modify the images to include Oracle JDK instead of Zulu OpenJDK.
 
-  1. Change the base image to install Oracle JDK instead of Zulu OpenJDK.  First, you'll need to update the base image ``Dockerfile``:
+  1. Change the base image to install Oracle JDK instead of Zulu OpenJDK by updating ``debian/base/Dockerfile``.
 
     .. sourcecode:: bash
 
-       FROM debian:jessie
-
-       ARG COMMIT_ID=unknown
-       LABEL io.confluent.docker.git.id=$COMMIT_ID
-       ARG BUILD_NUMBER=-1
-       LABEL io.confluent.docker.build.number=$BUILD_NUMBER
-
-       MAINTAINER partner-support@confluent.io
-       LABEL io.confluent.docker=true
-
-
-       # Python
-       ENV PYTHON_VERSION="2.7.9-1"
-       ENV PYTHON_PIP_VERSION="8.1.2"
-
-       # Confluent
-       ENV SCALA_VERSION="2.11"
-       ENV CONFLUENT_MAJOR_VERSION="3.3"
-       ENV CONFLUENT_VERSION="3.3.0-SNAPSHOT"
-       ENV CONFLUENT_DEB_VERSION="1"
-
-       # Zulu
-       ENV ZULU_OPENJDK_VERSION="8=8.15.0.1"
+       # In debian/base/Dockerfile
```

Comment:
Alternatively, we could also provide a ready-to-use `Dockerfile` template for Oracle JDK, e.g. as `debian/base/Dockerfile.oracleJDK`?